it/helpers/IntegrationTestConstants.scala

```diff
@@ -41,19 +44,29 @@ object IntegrationTestConstants {
     firstName = None,
     lastName = None,
     tradingName = Some("Vatmobile Taxi"),
-    organisationName = Some("Vatmobile Taxi LTD")
+    organisationName = Some("Vatmobile Taxi LTD"),
+    mandationStatus = nonMTDfB
   )
 
   val individual = CustomerDetails(
     firstName = Some("Pepsi"),
     lastName = Some("Mac"),
     tradingName = Some("PepsiCo"),
-    organisationName = None
+    organisationName = None,
+    mandationStatus = nonMTDfB
   )
 
   val individualJson: JsObject = Json.obj(
     "firstName" -> "Pepsi",
     "lastName" -> "Mac",
-    "tradingName" -> "PepsiCo"
+    "tradingName" -> "PepsiCo",
+    "mandationStatus" -> mandated
```

Comment:
Can you rename the val from inividualJson to individualMandatedJson please for clarity